internal/npm_install/npm_install.bzl

```diff
@@ -147,20 +174,22 @@ npm_install = repository_rule(
 def _yarn_install_impl(repository_ctx):
   """Core implementation of yarn_install."""
 
-  # Put our package descriptors in the right place.
-  repository_ctx.symlink(
-      repository_ctx.attr.package_json,
-      repository_ctx.path("package.json"))
+  node = repository_ctx.path(get_node_label(repository_ctx))
+  yarn = get_yarn_label(repository_ctx)
+
   if repository_ctx.attr.yarn_lock:
-      repository_ctx.symlink(
-          repository_ctx.attr.yarn_lock,
-          repository_ctx.path("yarn.lock"))
+    # Copy the file over instead of using a symlink since the lock file
```

Comment:
It seems like this means that running `bazel` will no longer update the checked-in lockfile. I think it's actually a feature today, that you don't have to manually run `yarn` to update the lockfile. Needs some discussion.
What do you think about moving the excluded_packages bit to a separate PR?

Reply:
i agree